TS03-Subset-Data-and-No-Data-Values-In-R.Rmd

```diff
@@ -162,119 +168,171 @@ write.csv(harMet15.09.11, file="Met_HARV_15min_2009_2011.csv")
 
 ```
 
-
+<div id="challenge" markdown="1">
 #Challenge: Subsetting & Plotting
-1.Plot precipitation for only July 2010 in the Harvard Forest
- 
+
+Plot precipitation data for the month of July 2010 in Harvard Forest.
+
+</div>
+
 ``` {r challenge-code-subsetting, include=TRUE, results="hide", echo=FALSE}
+
+#subset out data points from July 2010
 harMet15_July2010 <- subset(harMet15.09.11, datetime >= as.POSIXct('2010-07-01 00:00',
         tz = "America/New_York") & datetime <=
         as.POSIXct('2010-07-31 23:59', tz = "America/New_York"))
 
-#did it work?
+#view first and last rows of data.frame to ensure subset worked
 head(harMet15_July2010$datetime)
 tail(harMet15_July2010$datetime)
 
-#ploting precip in July
+#plot precip data for July 2010
 qplot (datetime, prec,
        data= harMet15_July2010,
-       main= "Precipitation in July 2010\nHarvard Forest",
+       main= "Precipitation: July 2010\nNEON Harvard Forest Field Site",
        xlab= "Date", ylab= "Precipitation (mm)")
 ```
 
 
-## Missing Data - Dealing with data gaps
+## Managing Missing Data: NoData values
```

Comment:
They are `NoData` until we talk :) It will take 5 mins to change them all at once!